.gitignore

```diff
@@ -1,9 +1,18 @@
+# General ignore patterns
 *
 !*.*
 !*/
+
+# Ignore build artifacts and temporary files
 *.o
 *.so
 *.a
 *.asv
 *~
-doc*/
+
+# Ignore Sphinx build output but include source files
+doc/build/
+doc/api_docs/cpp
```

Comment:
This is not reflecting the current directory path declared in the Doxyfile variable

![image](https://github.com/user-attachments/assets/e55c0879-2753-4883-9861-58acc177e097)

```suggestion
doc/api_docs/cpp/html
```

Reply:
Hi @pritkc , thank you for updating. This should still be changed though to reflect your Doxyfile variable:
```suggestion
doc/api_docs/cpp/html
```